src/sync/submitters/__tests__/uniqueKeysSubmitter.spec.ts

```diff
@@ -17,7 +18,7 @@ const imp4 = { ...imp1, keyName: 'k3', feature: 'anotherFeature' };
 
 function getParams(uniqueKeysCache: any) {
   const params = {
-    settings: { log: loggerMock, scheduler: { uniqueKeysRefreshRate: 200 }, core: { key: undefined} },
+    settings: { log: loggerMock, scheduler: { uniqueKeysRefreshRate: 200 }, core: { key: undefined }, storage: { type: STORAGE_MEMORY} },
```

Comment:
remove `, storage: { type: STORAGE_MEMORY}`. It seems that it is not necessary here